src/common/styles.c

```diff
@@ -829,10 +829,13 @@ void dt_styles_apply_style_item(dt_develop_t *dev,
         int legacy_ret = 1;
 
         if(module->legacy_params)
+        {
```

Comment:
Since the new `dt_op_legacy_params` returns 0 when module->legacy_params is NULL, the surrounding `if` isn't needed anymore.
(More later, I've only had a chance to look at the first four files so far.)

Reply:
Indeed, fixed.